opm/parser/eclipse/EclipseState/Grid/GridProperties.cpp

```diff
@@ -204,6 +204,16 @@ namespace Opm {
 
 
     template< typename T >
+    void GridProperties<T>::insertKeyword(const SupportedKeywordInfo& supportedKeyword) const {
+        int nx = m_eclipseGrid.getNX();
+        int ny = m_eclipseGrid.getNY();
+        int nz = m_eclipseGrid.getNZ();
+
+        m_properties.emplace( std::make_pair(supportedKeyword.getKeywordName() , GridProperty<T>( nx, ny , nz , supportedKeyword )));
```

Comment:
If you're using `emplace` you don't have to go via pair (that's the point). Save yourself some trouble:

`emplace( keywordname, GridProperty< T >( nx, ny, nz, supported )  )`

You might even be able to omit GridProperty< T > by using `{}`.